test/integration/template_admin_test.go

```diff
@@ -1560,12 +1561,19 @@ func TestClusterRole(t *testing.T) {
 	helmChartPath := testlib.ADMIN_HELM_CHART_PATH
 
 	t.Run("testEnabled", func(t *testing.T) {
-		output := helm.RenderTemplate(t, &helm.Options{}, helmChartPath,
+		output := helm.RenderTemplate(t, &helm.Options{
+			SetValues: map[string]string{
+				"admin.fullnameOverride": "full-name",
```

Comment:
Good point about testing `admin.namespace`.

I used `admin.fullnameOverride` as a short cut so I would not need to control the `cloud.cluster.name` and `admin.domain`. I am assuming that the `admin.fullname` template works correctly and not testing every combination of values that it could take, just the most convenient.